.github/scripts/validate_cli_tokens.py

```diff
@@ -0,0 +1,9 @@
+import requests
+import time
+
+time.sleep(60) #delay added as we need the CLI token validation workflow to complete
+
+github_url="https://api.github.com/repos/twilio/twilio-cli/actions/workflows/release-token-validation.yml/runs"
```

Comment:
nit: we already did implement similar logic as part of the script [trigger-and-wait](https://github.com/twilio/twilio-cli/blob/main/.github/scripts/trigger-and-wait.sh). We could have used that